app/src/forms/admin/routes.js

```diff
@@ -65,18 +390,149 @@ routes.get('/users/:userId', async (req, res, next) => {
 //Form componets help info
 //
 
+/**
+ * @openapi
+ * /admin/formcomponents/proactivehelp/object:
+ *  post:
+ *   tags:
+ *    - Admin
+ *   summary: create or update proactive help details form.io componen
+ *   description: This endpoint will create or update proactive help details form.io component
+ *   security:
+ *    - openId: []
+ *   requestBody:
+ *    required: true
+ *    content:
+ *      application/json:
+ *        schema:
+ *          $ref: '#/components/requestBodies/AdminReqProactiveHelpObject'
+ *   responses:
+ *    '200':
+ *      description: Success
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/responses/responseBody/AdminGetProactiveHelpObject'
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ *    '422':
+ *      $ref: '#/components/responses/Error/UnprocessableEntityDB'
+ */
 routes.post('/formcomponents/proactivehelp/object', async (req, res, next) => {
   await controller.createFormComponentsProactiveHelp(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/formcomponents/proactivehelp/{publishStatus}/{componentId}:
+ *  put:
+ *   tags:
+ *    - Admin
+ *   summary: update form.io component proactive help publish status
+ *   description: This endpoint will update form.io component proactive help publish status.
+ *   security:
+ *    - openId: []
+ *   parameters:
+ *      - in: path
+ *        name: publishStatus
+ *        schema:
+ *          type: boolean
+ *        description: Publish status of the Firm.io component.
+ *        required: true
+ *        example: true
+ *      - in: path
+ *        name: componentId
+ *        schema:
+ *          type: string
+ *        description: Id of the Form.io component.
+ *        required: true
+ *        example: true
+ *   responses:
+ *    '200':
+ *      description: Success
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/responses/responseBody/AdminProactiveHelpPublishStatus'
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ *    '422':
+ *      $ref: '#/components/responses/Error/UnprocessableEntityDB'
```

Comment:
For some reason this is showing up in the docs as "422 Unauthorized". It should be "403 Access Denied"?